src/popup/qrcode.css

```diff
@@ -125,3 +125,34 @@ html, body {
     color: var(--white-100);
   }
 }
+
+/* Copy button  */
```

Comment:
While this works, it does not look very nice. I would have suggested to style the elements according to (now obsolete but still used here) Firefox Photon style guide like https://web.archive.org/web/20220519062052/https://design.firefox.com/photon/components/message-bars.html

Also maybe a success message using https://github.com/TinyWebEx/MessageHandler (already included here) could have been used instead, which would integrate it more into the existing style.